src/app/core/views/Preferences/tabs/Plans/PlanSelector.tsx

```diff
@@ -23,7 +23,7 @@ export default function PlanSelector({ className = '' }: { className?: string })
   const priceButtons = subscription?.type === 'subscription' ? 'change' : 'upgrade';
 
   const [prices, setPrices] = useState<DisplayPrice[] | null>(null);
-  const [interval, setInterval] = useState<'month' | 'year'>('year');
+  const [interval, setInterval] = useState<'month' | 'year' | 'lifetime'>('year');
```

Comment:
Since types comes from SDK I'll simply reuse the SDK Interface like this:
`const [interval, setInterval] = useState<DisplayPrice['interval']>('year');`

Reply:
okay 👍, but the main idea that what I propose is to avoid use magic strings. 
Avoiding magic strings we reach more solid code, avoiding possible typos, change the values more safety.
In this case it is only one file and it is not complicated, but if it is used in more places in the code it can get complicated.